build.sh

```diff
@@ -32,7 +32,12 @@ bazel test -j 200 //... --test_tag_filters "$tag_filter" --experimental_executio
 # Make sure that Bazel query works.
 bazel query 'deps(//...)' > /dev/null
 # Check that we can load damlc in ghci
-da-ghci --data yes //:repl -e '()'
+GHCI_SCRIPT=$(mktemp)
+cat <<EOF > $GHCI_SCRIPT
+:main --help
+:quit
+EOF
+da-ghci --data yes //:repl -ghci-script $GHCI_SCRIPT -e '()'
```

Comment:
You might be able to get away with something like this instead:

```
da-ghci --data yes //:repl -ghci-script <(echo ':main --help' && echo ':quit') -e '()'
```

Reply:
Or use `echo -e ':main --help\n:quit'`, but I think that's harder to read.

Reply:
tbh I find both harder to read.

Reply:
Fair. I think I'm a bit allergic to `mktemp` without cleanup.

Reply:
@SamirTalwar Good point. @cocreature You could add `trap EXIT "rm -f $GHCI_SCRIPT"` below `mktemp` for cleanup (if that's the right spelling).

Reply:
[not important]

You could also do:

```shell
function ghci_script {
  echo <<EOF
:m DA.Cli.Damlc
:main --help
:quit
EOF
}

da-ghci --data yes //:repl -ghci-script <(ghci_script) -e '()'
```

Or you could just check this file in somewhere and use it. 🙂

Reply:
I actually tried using a temporary file with `<` and for some reason `ghci` didn’t seem to like that and I don’t really want to spend my time debugging this :slightly_smiling_face: